test/connect-cloud-health/ConnectHealth.test.js

```diff
@@ -251,13 +260,110 @@ describe('Connect Cloud Health test suite', function () {
         process.kill(process.pid, 'SIGTERM');
     });
     it('Readiness returns 503 OK and STOPPED on STOPPED', function (done) {
+        process.removeAllListeners('SIGTERM');
+        let cloudHealth = new index_1.HealthChecker();
+        const promiseone = () => new Promise((resolve, _reject) => {
+            setTimeout(resolve, 1);
+        });
+        let checkone = new index_1.ShutdownCheck("test1", promiseone);
+        cloudHealth.registerShutdownCheck(checkone);
+        const reqStub = {};
+        const nextStub = {};
+        const resStub = {
+            write: sinon_1.default.fake(),
+            end: function () {
+                let expectedStatus = 503;
+                let code = resStub.statusCode ? resStub.statusCode : 0;
+                code.should.equals(expectedStatus, `Should return: ${expectedStatus}, but returned: ${code}`);
+                let expectedBody = "{\"status\":\"STOPPED\",\"checks\":[{\"name\":\"test1\",\"state\":\"STOPPED\",\"data\":{\"reason\":\"\"}}]}";
+                sinon_1.default.assert.calledWith(resStub.write, expectedBody);
+                done();
+            }
+        };
+        process.once('SIGTERM', () => __awaiter(this, void 0, void 0, function* () {
+            yield setTimeout(() => __awaiter(this, void 0, void 0, function* () {
+                index_1.LivenessEndpoint(cloudHealth)(reqStub, resStub, nextStub);
```

Comment:
should this Readiness test be testing on a LivenessEndpoint?

Reply:
Done